src/sat/smt/euf_solver.cpp

```diff
@@ -349,6 +349,20 @@ namespace euf {
         si.uncache(literal(v, true));
     }
 
+    bool solver::decide(bool_var& var, lbool& phase) {
```

Comment:
This is only going to be enabled if there is a user-propagator. Otherwise, it is never used.